### ISSUE TITLE
Tiger-proxy certificate issue 

### ISSUE THREAD

**Issue by jisspala** [gematik/epa-deployment#28]
**Steps to Reproduce:**

Start all services using the command:`docker-compose -f dc-mocks.yml up` 

All service containers will show a healthy status, except for tiger-proxy. The health status of tiger-proxy is as follows:

```
"Health": {
  "Status": "unhealthy",
  "FailingStreak": 46,
  "Log": [
    {
      "Start": "2024-08-29T14:50:47.486744176+02:00",
      "End": "2024-08-29T14:50:47.530896422+02:00",
      "ExitCode": 1,
      "Output": "wget: can't connect to remote host: Connection refused\n"
    }
  ]
}
```
Visit https://localhost/ in a browser. The browser shows an SSL warning. Continue by selecting the "**Accept Risk**" option.

The connection suddenly closes, and the browser displays the message:
`"localhost unexpectedly closed the connection`."

Upon checking the service logs, the following messages appear:

```

tiger-proxy                | 2024-08-29T12:56:32.231Z  INFO 1 --- [orkerEventLoop7] o.b.jsse.provider.ProvTlsServer          : [server #10 @6de25193] accepting connection from (unknown):(unknown)
tiger-proxy                | 2024-08-29T12:56:32.235Z  INFO 1 --- [orkerEventLoop7] o.b.jsse.provider.ProvTlsServer          : [server #10 @6de25193] received fatal(2) certificate_unknown(46) alert
tiger-proxy                | 2024-08-29T12:56:32.235Z  INFO 1 --- [orkerEventLoop7] o.b.jsse.provider.ProvTlsServer          : [server #10 @6de25193] disconnected from (unknown):(unknown)
tiger-proxy                | 2024-08-29T12:56:32.236Z  WARN 1 --- [orkerEventLoop7] d.g.t.t.m.n.u.PortUnificationHandler     : TLS handshake failure:
tiger-proxy                | 
tiger-proxy                |  Client does not trust MockServer Certificate Authority for:[id: 0xd9edb655, L:/172.24.0.3:443 ! R:/172.24.0.1:53694]. See http://mock-server.com/mock_server/HTTPS_TLS.html to enable the client to trust MockServer Certificate Authority.
tiger-proxy                | 
tiger-proxy                | 2024-08-29T12:56:32.241Z  INFO 1 --- [orkerEventLoop3] o.b.jsse.provider.ProvTlsServer          : [server #11 @576dd0a3] accepting connection from (unknown):(unknown)
tiger-proxy                | 2024-08-29T12:56:32.247Z  INFO 1 --- [orkerEventLoop3] o.b.jsse.provider.ProvTlsServer          : [server #11 @576dd0a3] established connection with (unknown):(unknown)
tiger-proxy                | 2024-08-29T12:56:32.249Z  INFO 1 --- [orkerEventLoop3] o.b.jsse.provider.ProvTlsServer          : [server #11 @576dd0a3] disconnected from (unknown):(unknown)
```
**Note :**  We have checked the tiger-proxy certificates inside ca.p12 file  https://github.com/gematik/epa-deployment/blob/main/tiger-proxy/ca.p12. We found that the CA certificate expired on **Thu, 18 Jul 2024 12:55:12 GMT.**

We also tried using a self-signed certificate, but it did not work.

**Comment by sonerd** [comment]
Hey @jisspala ,

thanks for the info. We just a released a new version including updated certificate.
Problem should be resolved now.

Regards.

**Comment by jisspala** [comment]
Hi @sonerd ,

Thanks for your reply and the update. We have pulled the latest version and run it, but unfortunately, we are still experiencing the issue. Here is the log we are getting when we access https://localhost/ in Chrome

`tiger-proxy                | 2024-09-02T16:57:09.851Z  INFO 1 --- [orkerEventLoop9] o.b.jsse.provider.ProvTlsServer          : [server #31 @39575337] accepting connection from (unknown):(unknown)
tiger-proxy                | 2024-09-02T16:57:09.854Z  INFO 1 --- [orkerEventLoop9] o.b.jsse.provider.ProvTlsServer          : [server #31 @39575337] received fatal(2) certificate_unknown(46) alert
tiger-proxy                | 2024-09-02T16:57:09.855Z  INFO 1 --- [orkerEventLoop9] o.b.jsse.provider.ProvTlsServer          : [server #31 @39575337] disconnected from (unknown):(unknown)
tiger-proxy                | 2024-09-02T16:57:09.855Z  WARN 1 --- [orkerEventLoop9] d.g.t.t.m.n.u.PortUnificationHandler     : TLS handshake failure:
tiger-proxy                | 
tiger-proxy                |  Client does not trust MockServer Certificate Authority for:[id: 0x6075c6e5, L:/172.24.0.4:443 ! R:/172.24.0.1:32866]See http://mock-server.com/mock_server/HTTPS_TLS.html to enable the client to trust MocksServer Certificate Authority.
tiger-proxy                | 
tiger-proxy                | 2024-09-02T16:57:09.860Z  INFO 1 --- [orkerEventLoop3] o.b.jsse.provider.ProvTlsServer          : [server #32 @8236407] accepting connection from (unknown):(unknown)
tiger-proxy                | 2024-09-02T16:57:09.865Z  INFO 1 --- [orkerEventLoop3] o.b.jsse.provider.ProvTlsServer          : [server #32 @8236407] established connection with (unknown):(unknown)
tiger-proxy                | 2024-09-02T16:57:09.866Z  INFO 1 --- [orkerEventLoop3] o.b.jsse.provider.ProvTlsServer          : [server #32 @8236407] disconnected from (unknown):(unknown)
tiger-proxy                | 2024-09-02T16:57:15.239Z  INFO 1 --- [io-8080-exec-10] o.b.jsse.provider.PropertyUtils          : Found string system property [java.home]: /opt/java/openjdk
tiger-proxy                | 2024-09-02T16:57:15.245Z  INFO 1 --- [orkerEventLoop4] d.g.t.t.p.h.AbstractTigerRouteCallback   : Received HTTP GET / User-Agent: 'unirest-java/3.1.00' Request-Length: 0 bytes => http://vau-proxy-server:8080/
tiger-proxy                | 2024-09-02T16:57:15.250Z  INFO 1 --- [ver-Scheduler16] d.g.t.t.p.h.AbstractTigerRouteCallback   : Returning HTTP 404 Response-Length: 0 bytes
tiger-proxy                | 2024-09-02T16:57:30.289Z  INFO 1 --- [nio-8080-exec-2] o.b.jsse.provider.PropertyUtils          : Found string system property [java.home]: /opt/java/openjdk
tiger-proxy                | 2024-09-02T16:57:30.295Z  INFO 1 --- [orkerEventLoop7] d.g.t.t.p.h.AbstractTigerRouteCallback   : Received HTTP GET / User-Agent: 'unirest-java/3.1.00' Request-Length: 0 bytes => http://vau-proxy-server:8080/
tiger-proxy                | 2024-09-02T16:57:30.300Z  INFO 1 --- [rver-Scheduler8] d.g.t.t.p.h.AbstractTigerRouteCallback   : Returning HTTP 404 Response-Length: 0 bytes
`
